index.html

```diff
@@ -0,0 +1,891 @@
+          beginning, because inflation erases everything that came before it.
+          All the chaos and randomness of the primordial moment are swept
+          away, forever out of our view.
+        </p>
+        <p>
+          “If you trace your cosmic roots,” said Abraham Loeb, a
+          Harvard-Smithsonian astronomer who was not part of the team, “you
+          wind up at inflation.”
+        </p>
+
+        <div class="article-end">
+          A version of this article appears in print on
+          March 18, 2014, Section A, Page 1 of the New York
+          edition with the headline: Space Ripples Reveal Big Bang’s Smoking
+          Gun
+          <span>. <a href="http://www.nytreprints.com/">Order Reprints</a> |
+            <a href="https://www.nytimes.com/section/todayspaper">Today’s Paper</a>
+            |
+            <a href="https://www.nytimes.com/subscriptions/Multiproduct/lp8HYKU.html?campaignId=48JQY">Subscribe</a>
+          </span>
+        </div>
+
+        <div class="social-icons">
+          <div class="social-icons-bar">
+            <ul class="social-icons-list">
+              <li>
+                <button class="btn-user-controls read-btn">
+                  Read 615 Comments
+                </button>
+              </li>
+              <li>
+                <ul class="right-aligned-icons bottom">
+                  <li>
+                    <i class="fa fa-facebook social" aria-hidden="true"></i>
+                  </li>
+                  <li>
+                    <i class="fa fa-twitter social" aria-hidden="true"></i>
+                  </li>
+                  <li>
+                    <i class="fa fa-envelope social" aria-hidden="true"></i>
+                  </li>
+                  <li><i class="fa fa-share social" aria-hidden="true"></i></li>
+                </ul>
+              </li>
+              
+            </ul>
+          </div>
+        </div>
+      </article>
+    </div>
+    <section class="suggested-articles-section">
+      <div class="same-category-container">
+        <div class="category-title">
+          <hr />
+          <a
+            href="https://www.nytimes.com/section/science/space?action=click&module=MoreInSection&pgtype=Article&region=Footer&contentCollection=Space%20&%20Cosmos">More
+            in Space & Cosmos</a>
+        </div>
+        <div class="same-category-row-1">
+          <div class="suggested-article first-row-element">
+            <figure>
+              <img src="images/art1-row1.jpg" alt="Article thumbnail" />
+              <figcaption>
+                Image by Space Telescope Science Institute/NASA, via
+                Associated Press
+              </figcaption>
+            </figure>
+            <span class="suggested-article-title">The Interstellar Comet Has Arrived in Time for the
+              Holidays</span>
+            <span class="suggested-article-date">Dec. 7</span>
+          </div>
```

Comment:
Each `div` in 'More in Space & Cosmos' section  should contain link to other article.